test/test_commands.py

```diff
@@ -134,6 +135,12 @@ def test_reimport(self):
         # newer git versions don't append three dots after the commit hash
         assert output == expected or output == expected.replace(b'... ', b' ')
 
+    def test_validate(self):
+        output = run_command(
+            'validate', ['--input', REPOS2_FILE, '.'])
```

Comment:
The new `validate` command doesn't need / use the positional `path` argument and therefore shouldn't have it.